canary/canary.csproj

```diff
@@ -14,8 +14,11 @@
     <PackageReference Include="Microsoft.AspNetCore.SpaServices.Extensions" Version="3.1.3" />
     <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="3.1.3" />
     <PackageReference Include="Microsoft.EntityFrameworkCore.Sqlite" Version="3.1.3" />
-    <PackageReference Include="VRDR" Version="3.1.0-preview3" />
-    <!-- <ProjectReference Include="..\vrdr-dotnet\VRDR\DeathRecord.csproj" /> -->
+    <PackageReference Include="VRDR" Version="3.1.0-preview9" />

```

Comment:
I think you only need to reference VRDR.Messaging since that will pull in VRDR automatically.